### ISSUE TITLE
Stacked note scrolling requires multiple clicks when you're scrolling to the left

### ISSUE THREAD

**Issue by apatel762** [apatel762/backlinks-ssg#56]
to reproduce, scroll to the end of:
<http://0.0.0.0:8000/?stackedNotes=%2F2021-01-25T095448Z.html&stackedNotes=%2F2021-01-24T192821Z.html&stackedNotes=%2F2021-01-24T191134Z.html>

and click the backlink for the 'Apple' note.

it won't scroll all the way to the left to bring the note into view, but if you click a link to a note that's already open on the right hand side, scrolls the note completely into view

**Comment by apatel762** [comment]
technically this isn't a bug; you need multiple clicks to bring the note to the centre of the screen, but the scrolling is working as intended in the sense that it brings the note into view when you click it (albeit not into the **centre** of the view)

tl;dr: i can't be bothered to find a fix for this

**Comment by apatel762** [comment]
i should fix this after all. When there are lots of notes open, scrolling to the left completely falls apart because of the flex container getting bigger as the notes un-stack

not sure how to fix this though? keep scrolling until the note is in the center of the screen? seems a bit inefficient but could work
